projects/plugins/crm/includes/ZeroBSCRM.DAL3.Obj.Contacts.php

```diff
@@ -2072,6 +2072,26 @@ public function getContacts($args=array()){
 
                     }
 
+								// Checking and fixing name clashes between custom fields and linked objects
+								// (e.g. custom field with slug `company` and the company linked object)
+								// See: https://github.com/Automattic/zero-bs-crm/issues/3477
+								$this->add_name_clash_suffix_if_needed(
+									$resArr, // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+									array(
+										'tags',
+										'dnd',
+										'company',
+										'lastlog',
+										'owner',
```

Comment:
I've been looking at the PHP test failures related to ` Undefined array key "owner" in /tmp/wordpress-latest/src/wp-content/plugins/crm/includes/ZeroBSCRM.Edit.php on line 185`, and I was able to replicate the error in my logs locally by creating a new invoice and assigning it to a contact .

On logging `$data['invoiceObj']['contact']`, there was no `owner` key but there was an `owner_zbs-avoid-name-clash-temporary-fix` key. 

If I remove `owner` from the array passed in to `add_name_clash_suffix_if_needed`, I get no more warnings.
Should the `owner` have a prefix attached to it?

Reply:
Thanks for catching this! Probably it needs some more conditions to prevent adding suffix to Jetpack CRM fields, we want only to add the suffix to custom ones.

Reply:
Fixed in c8e6376. It is clumsy but it will always work because we are calling `fix_name_clash_if_needed` only in our export method. It would probably fail if we have two custom fields with the same key, what should not happen.